tests/persistence_tests/test_shamir_persistence.py

```diff
@@ -74,6 +77,7 @@ def test_abort(emulator: Emulator):
 
 @core_only
 def test_recovery_single_reset(emulator: Emulator):
+    assert emulator.client is not None
```

Comment:
leftovers?

Reply:
Right, deleted in [8e42975](https://github.com/trezor/trezor-firmware/pull/2580/commits/8e429753eddfb89ebf309788b0ad20d89b8307a9)